modules/manage/pages/raft-group-reconfiguration.adoc

```diff
@@ -0,0 +1,66 @@
+= Raft Group Reconfiguration
+:description: Learn how the Redpanda Raft group protocol provides consistency and availability during reconfiguration.
+
+Redpanda uses the glossterm:Raft[] consensus algorithm to manage cluster metadata. Redpanda stores this metadata as messages in a special topic in a Raft cluster instance, also known as a Raft group. In a Raft group, one member is the leader and the others are followers. A Raft group that stores cluster metadata is called a controller, and the leader of this group is the controller broker. The messages in the metadata topic contain the current set of brokers that are members of the group. 
+
+Unlike Kafka with Raft (KRaft), Redpanda also uses Raft as its replication protocol for the application data. Each topic glossterm:partition[] in Redpanda forms a Raft group comprised of its leader and follower brokers. Redpanda simplifies Kafka architecture because Raft handles both metadata management as well as data replication. See xref:get-started:architecture.adoc#raft-consensus-algorithm[How Redpanda Works] for a further explanation of how Raft manages data replication in Redpanda.
```

Comment:
```suggestion
Unlike Kafka with Raft (KRaft), Redpanda uses Raft not only for cluster metadata but also for the application data. Each topic glossterm:partition[] in Redpanda forms a Raft group comprised of its leader and follower brokers. Redpanda simplifies Kafka architecture because Raft handles both metadata management as well as data replication. See xref:get-started:architecture.adoc#raft-consensus-algorithm[How Redpanda Works] for a further explanation of how Raft manages data replication in Redpanda.
```